mistral/globals.cc

```diff
@@ -43,4 +43,13 @@ bool Arch::is_clkbuf_cell(IdString cell_type) const
     return cell_type == id_MISTRAL_CLKENA || cell_type == id_MISTRAL_CLKBUF;
 }
 
+void Arch::create_hps_mpu_general_purpose(int x, int y)
+{
+    BelId gp_bel = add_bel(x, y, id("cyclonev_hps_interface_mpu_general_purpose"), id("cyclonev_hps_interface_mpu_general_purpose"));
```

Comment:
No, if I do that the name is not recognized in the quartus-compatible verilog.  Unless you mean adding X(cyclonev_hps_interface_mpu_general_purpose) in there rather than f.i. X(MISTRAL_HPS_MPU_GENERAL_PURPOSE) ?